evm/p2p/lightchain.py

```diff
@@ -373,7 +367,6 @@ def peer_block_height(peer: LESPeer):
 
     async def get_canonical_block_by_number(self, block_number: int) -> BaseBlock:
         """Return the block with the given number from the canonical chain.
-
```

Comment:
Is this intentional or is your editor striping these blank lines.  

I think this is reducing readability of these docstrings.  Can you re-add them.

Reply:
No, this wasn't intentional. I'll re-add them.